webapp/app/assessment/fra/sections/forestAreaChange/actions.js

```diff
@@ -33,9 +33,6 @@ export const updateForestAreaChangeCell = (assessmentType, sectionName, tableNam
     R.when(R.always(rowIdx === 3), calculateMirrorValue(colIdx, rowIdx, 0, NumberUtils.add, state))
   )(state)
 
-  batch(() => {
-    dispatch(autosave.start)
-    dispatch(updateTableData(assessmentType, sectionName, tableName, data))
-    dispatch(postTableData(tableName, data))
-  })
+  dispatch(batchActions([autosave.start, updateTableData(assessmentType, sectionName, tableName, data)]))
```

Comment:
Similar actions are duplicated several times, can we somehow find a solution for less duplication?